packages/manager/apps/dedicated/client/app/dedicatedCloud/translations/Messages_de_DE.json

```diff
@@ -177,32 +179,6 @@
   "dedicatedCloud_tab_licences_active_spla_success": "Die Bestellung Nummer <a href='{{t0}}' target='_blank'>{{t1}}</a> für eine SPLA-Lizenz wurde erfolgreich durchgeführt.",
   "dedicatedCloud_tab_licences_active_spla_fail": "Bei der Anfrage zur Aktivierung der SPLA-Lizenz ist ein Fehler aufgetreten.",
   "dedicatedCloud_tab_backup": "Backup",
-  "dedicatedCloud_tab_veeam_on": "Das Backup aktivieren",
```

Comment:
Removed in https://github.com/ovh/manager/commit/61f53d4b6507158941a76defbc72b7302a4206f3#diff-9f3e5bab7895d880922f9ee2852ccdb8L182